namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-domain-events-preprod/resources/calculate-release-dates-api-topic.tf

```diff
@@ -0,0 +1,12 @@
+resource "kubernetes_secret" "calculate-release-dates-api-topic" {
+  metadata {
+    name      = "hmpps-domain-events-topic"
+    namespace = "calculate-release-dates-api-preprod"
```

Comment:
@yasinmustafa , Just to add here: This is writing secret to different namespace `calculate-release-dates-api-preprod`. If any of the  namespaces is migrated from `live-1` to `live`,  you have to manually copy the secrets from `hmpps-domain-events-preprod` to `calculate-release-dates-api-preprod`.